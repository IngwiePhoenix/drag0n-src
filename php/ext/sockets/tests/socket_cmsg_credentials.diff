006+ 
007+ Notice: Use of undefined constant SO_PASSCRED - assumed 'SO_PASSCRED' in /Users/Ingwie/Work/drag0n-php/ext/sockets/tests/socket_cmsg_credentials.php on line 18
008+ 
009+ Warning: socket_set_option() expects parameter 3 to be long, string given in /Users/Ingwie/Work/drag0n-php/ext/sockets/tests/socket_cmsg_credentials.php on line 18
010+ could not set SO_PASSCRED
006- int(5)
007- Array
008- (
009-     [name] => 
010-     [control] => Array
011-         (
012-             [0] => Array
013-                 (
014-                     [level] => %d
015-                     [type] => %d
016-                     [data] => Array
017-                         (
018-                             [pid] => %d
019-                             [uid] => %d
020-                             [gid] => %d
021-                         )
022- 
023-                 )
024- 
025-         )
026- 
027-     [iov] => Array
028-         (
029-             [0] => dread
030-         )
031- 
032-     [flags] => 0
033- )
034- bool(true)