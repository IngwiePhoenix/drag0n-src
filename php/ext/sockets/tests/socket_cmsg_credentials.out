creating send socket
resource(6) of type (Socket)
creating receive socket
resource(7) of type (Socket)
bool(true)

Notice: Use of undefined constant SO_PASSCRED - assumed 'SO_PASSCRED' in /Users/Ingwie/Work/drag0n-php/ext/sockets/tests/socket_cmsg_credentials.php on line 18

Warning: socket_set_option() expects parameter 3 to be long, string given in /Users/Ingwie/Work/drag0n-php/ext/sockets/tests/socket_cmsg_credentials.php on line 18
could not set SO_PASSCRED